packages/strapi-design-system/src/Select/SelectTags.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import CloseIcon from '@strapi/icons/CloseAlertIcon';
+import styled from 'styled-components';
+import { Tag } from '../Tag';
+import { Row } from '../Row';
+
+const SelectTagsWrapper = styled.div`
+  margin-bottom: ${({ theme }) => theme.spaces[1]};
+`;
+
+const SelectTag = styled(Tag)`
+  position: relative;
+  z-index: 1;
+  margin-left: ${({ theme }) => theme.spaces[1]};
+  margin-top: ${({ theme }) => theme.spaces[1]};
+`;
+
+export const SelectTags = ({ tags, onRemoveTag, disabled }) => {
+  const handleClick = (value) => {
+    if (disabled) return;
+
+    onRemoveTag(value);
+  };
+
+  return (
+    <SelectTagsWrapper>
+      <Row wrap="wrap">
+        {tags.map((tag) => (
+          <SelectTag
+            icon={<CloseIcon />}
+            aria-disabled={disabled}
+            onClick={() => handleClick(tag.value)}
+            tabIndex={-1}
+            key={`tag-${tag.value}`}
```

Comment:
this will work only if the tags are all different right?

Reply:
At some point they should be, if it happen to be problematic we can always make an update later to generate a unique id.

Reply:
What would be the value of having multiple time the same identifier in a list of options?  What is different from selecting an option "pizza" and an option "pizza"?

I personnally think they should be unique 😊